cumulusci/core/config/base_task_flow_config.py

```diff
@@ -72,3 +76,10 @@ def get_suggested_name(self, name, steps):
             return f'. Did you mean "{match_list[0]}"?'
         else:
             return ""
+
+
+from .base_task_flow_config import *  # noqa
+
+warn(
+    "Deprecation warning: Please import from .base_task_flow_config rather than .BaseTaskFlowConfig"
+)
```

Comment:
Huh? This is base_task_flow_config

Reply:
Merge error caused by renaming files. Fixed now.